diagnostic.md

```diff
@@ -73,13 +89,26 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "POST" "http://www.example.com/donuts/" \
+  --header "Content-Type: application/json" \
+  --data $"{
+    \"donut\": {
+      \"name": \"French Cruller\",
+      \"price\": \"$0.99\"
+    }
+  }
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (data) {
+  return $.ajax({
+    url: app.host + '/donuts/',
+    method: 'POST',
+    data: data,
```

Comment:
Close. But we know what the data is so in this case it would be:

```js
let createDonut = function () {
  return $.ajax({
    url: 'http://localhost:3000/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```